apps/crn-server/src/handlers/working-group/update-deliverables-handler.ts

```diff
@@ -29,19 +22,15 @@ export const workingGroupUpdateHandler =
     dataProvider: WorkingGroupDataProvider,
   ): EventBridgeHandler<WorkingGroupEvent, WorkingGroupPayload> =>
   async (event) => {
+    const workingGroupId = event.detail.resourceId;
     logger.info(
-      `Received ${event.detail.type} event for ${event.detail.payload.data?.title?.iv}`,
```

Comment:
I'm changing from `title` to `id` because the structure of the event is different in Squidex and Contentful.

In contentful we would get the title by doing something like `event.detail.fields.title['en-US']`. Since the advantage of displaying the title instead of the id is not significantly greater, I've changed to id, that we can get in the same way in both squidex and contentful.